Use non-executable stack to make selinux happier
Upstreamed to kdesupport,
r1204248 | rdieter | 2010-12-06 08:05:09 -0600 (Mon, 06 Dec 2010) | 2 lines

diff -ur qimageblitz-0.0.706674/blitz/asm_scale.S qimageblitz-0.0.706674-noexecstack/blitz/asm_scale.S
--- qimageblitz-0.0.706674/blitz/asm_scale.S	2007-08-13 20:49:30.000000000 +0200
+++ qimageblitz-0.0.706674-noexecstack/blitz/asm_scale.S	2008-02-07 04:23:47.000000000 +0100
@@ -1,3 +1,4 @@
+.section .note.GNU-stack,"",%progbits
 
 #if defined(__EMX__)
 /* Due to strange behaviour of as.exe we use this macros */
